mobile/src/libs/enums/notification/notification-type.enum.ts

```diff
@@ -0,0 +1,7 @@
+const NotificationType = {
```

Comment:
move it to libs/packages/notification/notification-type.enum.ts and re-export from  libs/packages/notification/notification.ts